sdks/java/io/tika/src/main/java/org/apache/beam/sdk/io/tika/TikaIO.java

```diff
@@ -60,85 +92,96 @@
 public class TikaIO {
 
   /**
-   * A {@link PTransform} that parses one or more files and returns a bounded {@link PCollection}
-   * containing one element for each sequence of characters reported by Apache Tika SAX Parser.
+   * A {@link PTransform} that matches and parses the files
+   * and returns a bounded {@link PCollection} of {@link ParseResult}.
    */
-   public static Read read() {
-     return new AutoValue_TikaIO_Read.Builder()
-        .setQueuePollTime(Read.DEFAULT_QUEUE_POLL_TIME)
-        .setQueueMaxPollTime(Read.DEFAULT_QUEUE_MAX_POLL_TIME)
+  public static ParseAll parseAll() {
```

Comment:
"SomethingAll" tends to refer in the Beam codebase to something that takes a PCollection of inputs, e.g. TextIO.readAll() reads a PCollection of filepatterns, whereas TextIO.read() reads a single filepattern. I suggest to rename this to TikaIO.parse().

Reply:
np, I'll then also rename parseFiles to parseAll, as it will be a bit more consistent with the 'All' naming pattern used everywhere in Beam, hope that will work for you as well